dotnet/csharp-load/Program.cs

```diff
@@ -0,0 +1,88 @@
+﻿
+namespace application;
```

Comment:
two naming nits.

1. Can we put this in a directory that makes it more evident that it is circumventing the SDK?  e.g. `csharp-grpc-load` or `csharp-protos-loadgen` or something?  We are going to be adding another load generator here soon that actually does exercise the SDK, so it would be good to make sure it's easy to differentiate between the two?
2. I'm out of date on my c#/dotnet but would it not be typical to have `Momento` as part of this namespace declaration?  I'm assuming it's similar to java package naming conventions.

Reply:
Sure I'll rename the namespace.

I'm in the middle of adding a momento client path as well though. Should be there in the update which is still a couple few hours away.

Reply:
okay... I had been hoping to do basically a line-for-line port of the loadgen from the other languages to exercise the client, so that they would be somewhat consistent and approachable from one of these to the next, but it sounds like what you are doing may supersede that...